.dir-locals.el

```diff
@@ -20,7 +20,8 @@
   (fill-column . 80)
   (c-file-style . "swift"))
  (c++-mode
-  (whitespace-style face lines indentation:space))
+  (whitespace-style face lines indentation:space)
+  (flycheck-clang-language-standard . "c++14"))
```

Comment:
Will this mess up people who don't have flycheck installed?

Reply:
@jrose-apple not as far as I know (I have read manual attempting to tell, given up, and tried here with a no-local-state emacs and it seems harmless)